pyccel/codegen/f2py.py

```diff
@@ -154,42 +157,68 @@ def create_shared_library(codegen,
     base_dirpath = os.getcwd()
     os.chdir(pyccel_dirpath)
 
-    if language == 'fortran':
+    # Name of shared library
+    if sharedlib_modname is None:
+        sharedlib_modname = module_name
+
+    sharedlib_folder = ''
+
+    if language == 'c':
+        wrapper_code = create_c_wrapper(sharedlib_modname, codegen)
```

Comment:
If we are not calling `f2py` when we generate C code, maybe we should change the name of this module?